ui/tests/acceptance/mfa-method-test.js

```diff
@@ -0,0 +1,285 @@
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { click, currentRouteName, currentURL, fillIn, visit } from '@ember/test-helpers';
+import authPage from 'vault/tests/pages/auth';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import ENV from 'vault/config/environment';
+import { Response } from 'miragejs';
+import { underscore } from '@ember/string';
+
+module('Acceptance | mfa-method', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  hooks.before(function () {
+    ENV['ember-cli-mirage'].handler = 'mfaConfig';
+  });
+  hooks.beforeEach(function () {
+    this.store = this.owner.lookup('service:store');
+    this.getMethods = () =>
+      ['Totp', 'Duo', 'Okta', 'Pingid'].reduce((methods, type) => {
+        methods.addObjects(this.server.db[`mfa${type}Methods`].where({}));
+        return methods;
+      }, []);
+    return authPage.login();
+  });
+  hooks.after(function () {
+    ENV['ember-cli-mirage'].handler = null;
+  });
+
+  test('it should display landing page when no methods exist', async function (assert) {
```

Comment:
same question as before, you'll know better than me but should we start out every test with assert.expect(x)?

Reply:
So it's only required by the lint rule if there are assertions outside of the main scope.
```js
test('foo', async function (assert) {
  assert.expect(1);
  
  this.server.get('/some-endpoint', () => {
    assert.true('endpoint is hit');
  });
});
```
I always used to add it for every test but haven't really in this codebase since no one else really used it until it was required by the lint updates.

Reply:
ah, that makes sense. Thanks for the clarification.